tests/testthat/test-getData.R

```diff
@@ -1,77 +1,113 @@
-                                                   fileEncoding = "windows-1252"))) > 2000)
-  } else { # linux or mac
+    testLoaded <- getData(name = "MAIA Humans CSV",
+                          options = dataOptions(sep = ";",
+                                                fileEncoding = "windows-1252")) %>%
+      passOnErrorMsg()
+    if (foundResource(testLoaded)) expect_true(nrow(testLoaded) > 2000)
+    
+    # more rows if newer windows
+    testLoaded <- getData(name = "IsoMedIta Humans 21-12-22 - CSV",
+                          options = dataOptions(sep = ";"))
+    if (foundResource(testLoaded)) expect_true(nrow(testLoaded) > 2000)
+  }
+  })
+
+test_that("Test getData() from csv if linux or mac", {
+  if (!Sys.info()["sysname"] == "Windows") { # linux or mac
     # no error without specific encoding
-    expect_true(nrow(getData(name = "MAIA Humans CSV",
-                             options = dataOptions(sep = ";"))) > 2000)
+    testLoaded <- getData(name = "MAIA Humans CSV",
+                          options = dataOptions(sep = ";")) %>%
+      passOnErrorMsg()
+    if (foundResource(testLoaded)) expect_true(nrow(testLoaded) > 2000)
     
     # less data with windows encoding
     testLoaded <-
       getData(name = "MAIA Humans CSV",
               options = dataOptions(sep = ";",
-                                    fileEncoding = "windows-1252"))
+                                    fileEncoding = "windows-1252")) %>%
+      passOnErrorMsg()
     
-    expect_true(nrow(testLoaded) > 700)
-    expect_true(nrow(testLoaded) < 800)
+    if (foundResource(testLoaded)) {
+      expect_true(nrow(testLoaded) > 700)
+      expect_true(nrow(testLoaded) < 800)
+    }
   }
+})
+
+test_that("Test getData() from csv", {
+  testLoaded <- getData(name = "CIMA Animals 29.05.2021 CSV",
+                        options = dataOptions(sep = ";")) %>%
+    passOnErrorMsg()
+  if (foundResource(testLoaded)) expect_true(nrow(testLoaded) > 4000)
   
-  expect_true(nrow(getData(name = "CIMA Animals 29.05.2021 CSV",
-                           options = dataOptions(sep = ";"))) > 4000)
-  
-  expect_true(nrow(getData(name = "CIMA Plants 29.05.2021 CSV",
-                           options = dataOptions(sep = ";"))) > 100)
-  
-  expect_true(nrow(getData(name = "SAAID_V.2.0_2023 Animals (CSV)")) > 2000)
-  
-  expect_true(nrow(getData(name = "Zanadamu CSV format",
-                           options = dataOptions(fileEncoding = "ISO-8859-1"))) > 200)
+  testLoaded <- getData(name = "CIMA Plants 29.05.2021 CSV",
+                        options = dataOptions(sep = ";")) %>%
+    passOnErrorMsg()
+  if (foundResource(testLoaded)) expect_true(nrow(testLoaded) > 100)
   
-  if (isOldROnWindows()) {
-    expect_error(getData(name = "Isotopic measurements in CSV format"))
-  } else {
-    expect_true(nrow(getData(name = "Isotopic measurements in CSV format")) > 2000)
-  }
+  testLoaded <- getData(name = "SAAID_V.2.0_2023 Animals (CSV)") %>%
```

Comment:
Should the dataset still be included in the test, even if it no longer exists?
We could delete it or change it to the the `SAAID_V.2.0_2023_plants.csv` data set.
But be aware that there are problems with this data set:

```console
> getData(name = "SAAID_V.2.0_2023_plants.csv")
Encoding: 'UTF-8'.

Error in getData(name = "SAAID_V.2.0_2023_plants.csv") : 
  Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  more columns than column names
 for resource with name 'SAAID_V.2.0_2023_plants.csv', Please check dataOptions() for this resource
```